assets/cubyz/blocks/cactus_flower.zig.zon

```diff
@@ -9,7 +9,7 @@
 	.alwaysViewThrough = true,
 	.absorbedLight = 0x121012,
 	.model = "cubyz:cactus_flower",
-	.rotation = .log,
+	.rotation = .log_old,
```

Comment:
Interesting, so here we have a use-case of log_old that we won't remove after the mgirations are available.
That means we must keep the rotation mode, so it should get a better name than just `log_old`